newIDE/app/src/GameDashboard/GameDashboardCard.js

```diff
@@ -0,0 +1,657 @@
+        <Text color="secondary" noMargin size="body-small">
+          <Trans>Last edited:</Trans>
+        </Text>
+        <Text color="secondary" noMargin size="body-small">
+          {i18n.date(game.updatedAt * 1000)}
+        </Text>
+      </LineStackLayout>
+    ) : null;
+
+  const renderStorageProvider = (i18n: I18nType) => {
+    const icon = itemStorageProvider ? (
+      itemStorageProvider.renderIcon ? (
+        itemStorageProvider.renderIcon({
+          size: 'small',
+        })
+      ) : null
+    ) : (
+      <WarningRound />
+    );
+    const name = itemStorageProvider ? (
+      i18n._(itemStorageProvider.name)
+    ) : isCurrentProjectOpened ? (
+      <Trans>Project not saved</Trans>
+    ) : (
+      <Trans>Project not found</Trans>
+    );
+
+    return (
+      <Line noMargin alignItems="center">
+        {icon && (
+          <>
+            {icon}
+            <Spacer />
+          </>
+        )}
+        <Text noMargin color="secondary">
+          {name}
+        </Text>
+      </Line>
+    );
+  };
+
+  const renderThumbnail = () => (
+    <GameThumbnail
+      gameName={gameName}
+      gameId={game ? game.id : undefined}
+      thumbnailUrl={gameThumbnailUrl}
+      background="light"
+      width={
+        isMobile
+          ? undefined
+          : // On medium/large screens, adapt the size to the width of the window.
+            Math.min(245, Math.max(130, window.innerWidth / 4))
+      }
+    />
+  );
+
+  const buildOpenContextMenu = (
+    i18n: I18nType,
+    projectsList: FileMetadataAndStorageProviderName[]
+  ): Array<MenuItemTemplate> => {
+    const actions = [];
+    if (projectsList.length > 1) {
+      actions.push(
+        ...[
+          ...projectsList.map(fileMetadataAndStorageProviderName => {
+            return {
+              label: getProjectItemLabel(
+                fileMetadataAndStorageProviderName,
+                storageProviders,
+                i18n
+              ),
+              click: () => onOpenProject(fileMetadataAndStorageProviderName),
+            };
+          }),
+          { type: 'separator' },
+          {
+            label: i18n._(t`See all in the game dashboard`),
+            click: game ? () => onOpenGameManager({ game }) : undefined,
+          },
+        ]
+      );
+    }
+
+    return actions;
+  };
+
+  const renderAdditionalActions = () => {
+    return (
+      <ElementWithMenu
+        element={
+          <IconButton size="small" disabled={disabled}>
+            <ThreeDotsMenu />
+          </IconButton>
+        }
+        buildMenuTemplate={(i18n: I18nType) => {
+          const actions = [];
+
+          // Close action
+          if (isCurrentProjectOpened) {
+            actions.push({
+              label: i18n._(t`Close project`),
+              click: async () => {
+                await askToCloseProject();
+              },
+            });
+          }
+
+          // Management actions.
+          if (projectsList.length < 2) {
```

Comment:
I stumbled upon this condition, I was like "why 2" ? 😅 
So I recommend `=== 1` instead